src/components/RichText/_rich-text.scss

```diff
@@ -60,4 +95,40 @@
   td {
     font-size: var(--body-md);
   }
+
+  > :last-child {
+    margin-bottom: 0;
+  }
+}
+
+.cc-rich-text-snippet {
+  @extend %body-large;
+  line-height: var(--body-line-height-sm);
+
+  p {
+    @extend %body-regular;
+  }
+
+  ol,
+  ul {
+    @extend %list-base;
+    margin-bottom: calc(2 * var(--space-unit));
+  }
+
+  > :last-child {
+    margin-bottom: 0;
+  }
+}
+
+.cc-text-snippet__title,
```

Comment:
is this selector correct? feels wrong/like it shouldn't be here...

Reply:
I know what you mean but I felt it made less sense to keep _text.scss for a single rule.

I originally attempted to include the [title](https://github.com/wellcometrust/corporate-components/blob/10e7ffa275028b500d0ac67c4c4ab2b940f356b0/src/components/Text/Text.tsx#L25) as a child node of `<RichText>` to keep everything in the same place but was having TypeScript issues

Reply:
Thinking about it, it now seems like `<Text>` as a component might be redundant. Can you think of any reasons why it needs to be kept?